tests/atoms/gaia_apps.js

```diff
@@ -372,13 +372,13 @@ var GaiaApps = {
          .query('AppWindowManager.getActiveWindow').getTopMostWindow();
     }
 
-    let origin = app.origin;
-    console.log('app with origin \'' + origin + '\' is displayed');
+    console.log('app with origin and manifest url \'' + app.origin + ', ' + app.manifest_url + '\' is displayed');
```

Comment:
If you want to log both origin and manifest_url (which seems a bit overkill), then I'd prefer it something like this:
console.log('app with origin \'' + app.origin  + '\'and manifest url \'' + app.manifest_url + '\' is displayed');